.github/workflows/run-tests.yml

```diff
@@ -4,6 +4,8 @@ on:
     branches-ignore:
       - main
       - development
+env:
+  REACT_APP_INFURA_URL: https://kovan.infura.io/v3/number123
```

Comment:
this line is added to fix the unit test.

the lib is expecting an url:
https://github.com/NoahZinsmeister/web3-react/blob/v6/packages/network-connector/src/index.ts#L32